runCI/action.yml

```diff
@@ -0,0 +1,61 @@
+name: 'runCI Action'
+
+description: 'Action to run CI builds'
+
+inputs:
+  PROJECT_NAME:
+    required: true
+  CC:
+    required: true
+  DOCKER_IMAGE:
+    required: true
+  TPM2TOOLS_BRANCH:
+    required: false
+  TPM2TSS_BRANCH:
+    required: false
+  BASE_REF:
+    required: false
+  SCANBUILD:
+    required: false
+  TEST_TCTI_CONFIG:
+    required: false
+  WITH_CRYPTO:
+    required: false
+  ENABLE_COVERAGE:
+    required: false
+  GEN_FUZZ:
+    required: false
+  CXX:
+    required: false
+  GIT_FULL_CLONE:
+    required: false
+  MAKE_TARGET:
+    required: false
+  ENABLE_FUZZING:
+    required: false
+  TPM2_TSS_VERSION:
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Launch Container
+      env:
+        TPM2TOOLS_BRANCH: ${{ inputs.TPM2TOOLS_BRANCH }}
+        TPM2TSS_BRANCH: ${{ inputs.TPM2TSS_BRANCH }}
+        CC: ${{ inputs.CC }}
+        DOCKER_IMAGE: ${{ inputs.DOCKER_IMAGE }}
+        PROJECT_NAME: ${{ inputs.PROJECT_NAME }}
+        BASE_REF: ${{ inputs.BASE_REF }}
+        SCANBUILD: ${{ inputs.SCANBUILD }}
+        TEST_TCTI_CONFIG: ${{ inputs.TEST_TCTI_CONFIG }}
+        WITH_CRYPTO: ${{ inputs.WITH_CRYPTO }}
+        ENABLE_COVERAGE: ${{ inputs.ENABLE_COVERAGE }}
+        GEN_FUZZ: ${{ inputs.GEN_FUZZ }}
+        CXX: ${{ inputs.CXX }}
```

Comment:
Do we ever need CXX nothing in our project is C++, could this be a leftover/cruft thing?